portal-impl/src/com/liferay/portal/util/PropsValues.java

```diff
@@ -181,14 +181,6 @@
 	public static final String AUTH_LOGIN_PORTLET_NAME = PropsUtil.get(
 		PropsKeys.AUTH_LOGIN_PORTLET_NAME);
 
-	/**
```

Comment:
@jeffwu0724 For all the properties you removed here, you should also remove them from portal.properties and add them to VerifyProperties. Please do:
1. Check all the properties removed here on master, if the usages are all from deprecated methods/classes, remove the deprecated methods/classes first, then remove the properties.
2. Send me a pull for 1 to remove properties.
3. Rebase the changes in this pull, send me a pull to remove deprecate as normal.